nohup.out

```diff
@@ -0,0 +1,84 @@
+sudo: apt-mirror: command not found
```

Comment:
Oh actually, please remove this file. I suggest not running `git add .` so that doesnt happen next time.